core/src/main/java/sem/group15/bubblebobble/core/objects/Gravity.java

```diff
@@ -49,7 +49,7 @@ public void update(float elapsed) {
     public void handleCollision(GameObject other) {
         if (other instanceof Floor) {
             float MAX_DIFF_LANDING = 10f;
-            if (between(overlapBottom(other), 0, MAX_DIFF_LANDING) && speedY < 0) {
+            if (getTop() < Gdx.graphics.getHeight() && between(overlapBottom(other), 0, MAX_DIFF_LANDING) && speedY < 0) {
                 setBottom(other.getTop());
```

Comment:
watch out for to long lines, it is now set to 120, maybe put an and statement on a new line <layout>